docs/docusaurus/docs/guides/connecting_to_your_data/fluent/database/how_to_connect_to_sql_data.md

```diff
@@ -76,8 +74,8 @@ If you include a password as plain text in your connection string when you defin
 
 Creating a SQL Datasource is as simple as providing the `add_sql(...)` method a `name` by which to reference it in the future and the `connection_string` with which to access it.
 
-```python title="Python code"
-datasource = context.sources.add_sql(name="my_datasource", connection_string=sql_connection_string)
+
```

Comment:
@TrangPham There is a slight inconsistency between the snippet displayed in the document dynamically and another example appearing in the document (statically) -- in terms of the database driver.  Please open the Netlify view in order to see this.  Thanks!